README.md

```diff
@@ -10,15 +10,18 @@ For å installere dependencies:
 npm install
 ```
 
+For å få tilgang til alle dependencies må man generere en personal access-token som legges til i `.npmrc` på ~. Denne genereres via Github, trenger tilgangen `read:packages`, og må autentiseres via NAV.
```

Comment:
Vi kan med fordel også lenke til GitHub sin dokumentasjon også på dette 👍 

https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-to-github-packages